bindings/Sofa/src/SofaPython3/Sofa/Components/CMakeLists.txt

```diff
@@ -17,5 +17,5 @@ SP3_add_python_module(
         DESTINATION  Sofa
         SOURCES      ${SOURCE_FILES}
         HEADERS      ${HEADER_FILES}
-        DEPENDS      Sofa.Core Sofa.Helper Sofa.SimulationCore SofaPython3::Plugin
+        DEPENDS      Sofa.Simulation.Core SofaPython3::Plugin
```

Comment:
Should be propagated to [Bindings.SofaConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Sofa/Bindings.SofaConfig.cmake.in)